src/couch_file.erl

```diff
@@ -531,27 +537,96 @@ find_header(Fd, Block) ->
     {ok, Bin} ->
         {ok, Bin};
     _Error ->
-        find_header(Fd, Block -1)
+        find_header_pread2(Fd, Block -1)
     end.
 
 load_header(Fd, Block) ->
     {ok, <<1, HeaderLen:32/integer, RestBlock/binary>>} =
         file:pread(Fd, Block * ?SIZE_BLOCK, ?SIZE_BLOCK),
-    TotalBytes = calculate_total_read_len(5, HeaderLen),
```

Comment:
Undo the part of this PR that renames calculate_total_read_len to total_read_size. Its not a behavior change and shouldn't be included with this change as it just makes things harder to comprehend when looking through the history. If you feel strongly about it we can open another PR with this change for a separate readability improvement.